tests/resources/response/domain/check.xml

```diff
@@ -10,7 +10,7 @@
                     <domain:name avail="1">eppdev.com</domain:name>
                 </domain:cd>
                 <domain:cd>
-                    <domain:name avail="0">eppdev.net</domain:name>
+                    <domain:name avail="false">eppdev.net</domain:name>

```

Comment:
This should remain "0", right?

Reply:
Why? The spec allows both forms, seems nice to exercise deserialization for both.

Reply:
Ah, yes. I misread the RFC